ui/packages/app/src/components/Accordion/Accordion.svelte

```diff
@@ -5,7 +5,7 @@
 	export let visible: boolean = true;
 	export let open: boolean = true;
 
-	let _open = open;
+	$: _open = open;
```

Comment:
Why did this ever exist in the first place? It is just assigning the value to an intermediary variable? What happens if you just remove the `_open` and use `open` directly?

Reply:
ah.